src/test/java/net/sf/jabref/importer/fileformat/MedlineImporterTest.java

```diff
@@ -0,0 +1,93 @@
+/**
+ *
+ */
+package net.sf.jabref.importer.fileformat;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Assert;
+
+/**
+ * @author Daniel Mair/Bruehl
+ *
+ */
+public class MedlineImporterTest {
+
+    MedlineImporter medlineImp;
```

Comment:
Do not use abbreviations. Either use `medlineImporter` or `importer` as the context is clear by being in the `MedlineImporterTest` class.